.yamato/com.unity.ml-agents-test.yml

```diff
@@ -57,6 +56,12 @@ all_package_tests:
 {% for package in packages %}
   {% for editor in test_editors %}
     {% for platform in test_platforms %}
+
+{% if editor.enableCodeCoverage %}
+  {% capture coverageOptions %} --enable-code-coverage --code-coverage-options 'generateHtmlReport;assemblyFilters:+{{ package.assembly }}'{% endcapture %}
```

Comment:
No string interpolation in liquid. See https://stackoverflow.com/questions/48074046/how-to-interpolate-in-liquid-shopify and https://github.com/Shopify/liquid/wiki/Liquid-for-Designers#variable-assignment